Pipfile

```diff
@@ -5,7 +5,7 @@ name = "pypi"
 
 [packages]
```

Comment:
I would recommend installing jinja2 >= 2.10.1 just to get ahead of any future problems; see https://www.exploit-db.com/exploits/46386

Reply:
I see this is already listed in https://github.com/arXiv/arxiv-search/network/alerts; we might as well just take care of all of these at once.

Reply:
Fixed in 4c0993701ef55ed44981f90825c1c562ed62a5c8

Reply:
Had up to upgrade `elasticsearch`, too